include/silo_api/request_handler_factory.h

```diff
@@ -16,10 +16,13 @@ namespace silo_api {
 class SiloRequestHandlerFactory : public Poco::Net::HTTPRequestHandlerFactory {
   private:
    silo_api::DatabaseMutex& database;
-   const RuntimeConfig runtime_config;
+   const silo::config::RuntimeConfig runtime_config;
```

Comment:
Why are we not `using` the runtime config here?

Reply:
As it is a header file, I would prefer not to